venv/Lib/site-packages/PIL/ImageMath.py

```
@@ -0,0 +1,368 @@
+
+    scan(compiled_code)
+    out = builtins.eval(expression, {"__builtins": {"abs": abs}}, args)
+    try:
+        return out.im
+    except AttributeError:
+        return out
+
+
+def eval(
+    expression: str,
+    _dict: dict[str, Any] = {},
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)